postgresql/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM index.docker.io/fedora
+FROM fedora:22
 MAINTAINER Pavel Raiskup <praiskup@redhat.com>
 
 ENV container="docker"
 
-RUN yum -y --setopt=tsflags=nodocs install postgresql-server \
-    && yum -y --setopt=tsflags=nodocs reinstall glibc-common \
-    && yum -y --setopt=tsflags=nodocs clean all --enablerepo='*'
+RUN dnf -y --setopt=tsflags=nodocs install postgresql-server \
+    && dnf -y --setopt=tsflags=nodocs reinstall glibc-common \
+    && dnf -y --setopt=tsflags=nodocs clean all --enablerepo='*'
 
```

Comment:
It does but it is specific to postgres. This pull request is an effort to move the entire branch to dnf and to use fedora 22.